### ISSUE TITLE
Check the availability of Weblogo in PBstat.py

### ISSUE THREAD

**Issue by HubLot** [pierrepo/PBxplore#60]
For now, when launching PBstat.py whereas Weblogo software is not present, the script is launched anyway and just an error is printed, which could be easily missed.
Plus, in the README of tool, it states that Weblogo is optionnal.
From the run_demo2.sh script:

```
> ../PBstat.py -f psi_md_traj_all.PB.count -o psi_md_traj --map --neq --logo
Index of first residue is: 1
wrote psi_md_traj.PB.map.png

wrote psi_md_traj.PB.Neq
wrote psi_md_traj.PB.Neq.png

ERROR: /bin/sh: weblogo : command not found

ERROR: exit code != 0
exit code: 127
```

One solution could be to add a callback to the parsing of `--logo` option to test if Weblogo is present.
Since the README states weblogo is optional, I don't know which solution is better when weblogo is not present and the option ``--logo`` is set:
 - prevent the script to run
 - run the script but do not perform the logo part.

**Comment by alexdb27** [comment]
excellent idea.

**Comment by jbarnoud** [comment]
Fixed by @HubLot  with PR #61.
